packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -323,6 +370,815 @@ export class FabricTestLedgerV1 implements ITestLedger {
+      this.log.error(`populateFile() crashed: `, error);
+      throw new Error(`${fnTag} Unable to run transaction: ${error.message}`);
+    }
+  }
+
+  public setContainer(id: string): void {
+    const fnTag = "FabricTestLedgerV1#setContainer()";
+    let container;
+    try {
+      const docker = new Docker();
+      container = docker.getContainer(id);
+      this.container = container;
+      this.containerId = id;
+    } catch (error) {
+      throw new Error(`${fnTag} cannot set container.`);
+    }
+  }
+
+  public async addExtraOrgs(): Promise<void> {
+    const fnTag = `FabricTestLedger#addOrgX()`;
+    const { log } = this;
+    if (!this.extraOrgs) {
+      throw new Error(`${fnTag}: there are no extra orgs`);
+    }
+
+    log.debug(`
+    Adding ${this.extraOrgs.length} orgs`);
+    for (let i = 0; i < this.extraOrgs.length; i++) {
+      await this.addOrgX(
+        this.extraOrgs[i].path,
+        this.extraOrgs[i].orgName,
+        this.extraOrgs[i].orgChannel,
+        this.extraOrgs[i].certificateAuthority,
+        this.extraOrgs[i].stateDatabase,
+        this.extraOrgs[i].port,
+      );
+    }
+  }
+  // req: AddOrganizationFabricV2Request
+  // returns promise <AddOrganizationFabricV2Response>
+  public async addOrgX(
+    addOrgXDirectoryPath: string,
+    orgName: string,
+    channel = "mychannel",
+    certificateAuthority: boolean,
+    database: string,
+    peerPort = "11051",
+  ): Promise<void> {
+    const fnTag = `FabricTestLedger#addOrgX()`;
+    const { log } = this;
+    log.debug(`
+    Adding ${orgName} on ${channel}, with state database ${database}. 
+    Certification authorithy: ${certificateAuthority}.
+    Default port: ${peerPort}
+    Path to original source files: ${addOrgXDirectoryPath}`);
+
+    if (certificateAuthority) {
+      throw new Error("Adding orgs with CA enabled is not currently supported");
+    }
+
+    if (this.stateDatabase !== database) {
+      log.warn(
+        "Adding an organization with a different state database than org1 and org2",
+      );
+    }
+
+    const peerPortNumber = Number(peerPort);
+    const mspId = orgName + "MSP";
+
+    const ssh = new NodeSSH();
+    const sshConfig = await this.getSshConfig();
+    await ssh.connect(sshConfig);
+    log.debug(`SSH connection OK`);
+
+    try {
+      //Avoid well-known ports
+      if (peerPortNumber < 1024) {
+        throw new Error(`${fnTag} Invalid port, port too small`);
+      }
+      const couchDbPort = Math.abs(peerPortNumber - 1067).toString();
+      const caPort = (peerPortNumber + 3).toString();
+
+      temp.track();
+
+      //const tmpDirPrefix = `hyperledger-cactus-${this.className}`;
+      //const tmpDirPath = temp.mkdirSync(tmpDirPrefix);
+      /*
+      const tmpDirPrefix = `hyperledger-cactus-${this.className}`;
+      const tmpDirPath = temp.mkdirSync(tmpDirPrefix);
+      const tmpDirPathDocker = temp.mkdirSync(
+        path.join(tmpDirPrefix, "docker"),
+      );
+    */
+
+      //
+
```

Comment:
@RafaelAPB I recommend dropping the commented code if no longer needed. (assumption is that this commented code was a dead end during the implementation phase that was reasonable to be kept handy while it was uncertain whether it would be needed or not, but if in the end it turned out that it wasn't, then I'd prefer if it just went away to make the rest of the code around it easier to read). 
It is up to your discretion to decide if this is indeed the case or if there's real value in keeping the comment around for future reference (and if you say that that is the case then I will happily stand down of course)

```suggestion
```